inductiva/_cli/cmd_tasks/list.py

```diff
@@ -19,15 +21,37 @@ def list_tasks(args):
 
     table_dict = tasks.to_dict(task_list)
 
+    emph_formatter = inductiva.get_ansi_formatter()
+
+    def color_formater(status):
+        if status == models.TaskStatusCode.SUCCESS:
+            return emph_formatter(status, utils.format_utils.Emphasis.GREEN)
+        elif status in [
```

Comment:
Why elif here ? Can't we print success green and else red?

Reply:
There are status such as `SUBMITED` to which no colors are added